frontend/src/pages/home.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from "react";
+import '../css/pages/home.scss';
+import intro_img from '../images/pixeltrue-jogging 1.png';
+import BlogCard from '../components/BlogCard'
+import GoogleAd from '../images/GoogleAds.png'
+import Microsoft from '../images/Microsoft.png'
+import Model from '../images/model.png'
+import Amazon from '../images/amazon.png'
+import Figma from '../images/figma.png'
+import Triangle from '../images/Graphics.svg'
+
+class Home extends Component {
+    render() {
+      return (
+          <>
+            <div style={{background:"#FFFDF6"}}>
+                <div className="intro">
+                    <div className="grid-container grid-x grid-margin-x">
+                        <div className="cell small-7">
+                            <h1>
+                                Made for all, by women.
+                            </h1>
+                            <p>
+                                An inclusive space for all. A place where we understand, 
+                                acknowledge and leverage diversity by creating an inclusive 
+                                ecosystem powered by content, culture and community.
+                            </p>
+                            <br/>
+                            <p>
+                                Do you have a diversity centric organization or special interest group?
+                            </p>
+                            <br/>
+                            <a className="button__big" style={{marginRight:"10px"}}>
+                                Write a story
+                            </a>
+                            <a className="button__big button__big--hollow">
+                                Write a story
+                            </a>
+                        </div>
+                        <div className="cell medium-5">
+                            <img src={intro_img} alt="Feminist Bible" className="photo"/>
+                        </div> 
+                    </div>
+                </div>
+            </div>
+            <div style={{background:"linear-gradient(180deg, #FFFDF6 0%, #FFEBF3 100%)"}}>
+                <div className="importanceOfDiversity ">
+                    <div className="cell small-7 large-offset-1">
```

Comment:
offsets are not required anywhere for the homepage. Kindly remove all the offsets.

Reply:
Okay, got it 👍

Reply:
Done 👍